lib/paper_trail/record_trail.rb

```diff
@@ -406,6 +435,23 @@ def touch_with_version(name = nil)
       @record.save!(validate: false)
     end
 
+    # Like the `update_column` method from `ActiveRecord::Persistence`, but also
+    # creates a version to record those changes.
+    def update_column(name, value)
+      update_columns(name => value)
+    end
+
+    # Like the `update_columns` method from `ActiveRecord::Persistence`, but also
+    # creates a version to record those changes.
+    def update_columns(attributes)
+      changes = {}
+      attributes.each do |k, v|
+        changes[k] = [@record[k], v]
+      end
```

Comment:
Let's add a comment somewhere here explaining that we are constructing a hash that is compatible with the hash returned by `ActiveModel::Dirty#changes`.